sfge/src/main/java/com/salesforce/graph/ops/ApexPathUtil.java

```diff
@@ -66,8 +66,8 @@ public static ApexPathRetrievalSummary summarizeForwardPaths(
         BlockStatementVertex blockStatement =
                 method.getOnlyChildOrNull(ASTConstants.NodeType.BLOCK_STATEMENT);
         if (blockStatement == null) {
-            // Default constructors don't have a block statement. Create a synthetic one
-            // TODO: Consider adding this to the graph instead
+            // If a method doesn't have a block statement, we'll create a synthetic one in memory.
```

Comment:
In a perfect world, the changes related to this bugfix would have allowed us to simply delete this `if` entirely. However, that seems to cause at least one weird test failure. So I'm leaving this block in here for now, and we can make more progress towards fully removing it in the future.